kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -525,6 +525,20 @@ public void testGetCaseInstancesByStatusSorting() {
         assertEquals(hrCaseId, caseInstances.get(0).getCaseId());
         assertEquals(claimCaseId, caseInstances.get(1).getCaseId());
     }
+    
```

Comment:
Maybe it would good to add another test in this class for calling the new overloaded method `getCaseInstancesOwnedBy`